Internal/Vehicle/VehicleMP4.as

```diff
@@ -9,13 +9,34 @@ class CSceneVehicleVisInner
 	CSceneVehicleVisInner(CGamePlayer@ player)
 	{
 		@m_player = cast<CTrackManiaPlayer>(player);
+		if (m_player !is null) {
+			@AsyncState = CSceneVehicleVisState(Dev::GetOffsetNod(m_player, 0x2b8));
```

Comment:
Yes -- the pointers are the same as via the vehicle vis mgr. (That's actually how I found it -- searching for the pointer in the Player structure.)
Also, 0xc bytes (ish) before or after this is the vehicle/entity id.

Reply:
Note: there appears to be a similar pointer in tm2020's `CSmPlayer` via the offsets:

- 0x10E0 (`.Score` offset + 0xA8)
- 0x1A0

Not sure what I'm looking at via that pointer path, though. Figured I'd mention as it might be more efficient than the way it's done for TMNEXT atm.